Headlines/src/ScreenSwitcher/FadeTransition.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  FadeTransition.swift
+//  Canillitapp
+//
+//  Created by Ezequiel Becerra on 11/8/17.
+//  Copyright © 2017 Ezequiel Becerra. All rights reserved.
+//
+
+import UIKit
+
+class FadeTransition: NSObject, UIViewControllerAnimatedTransitioning {
+    func transitionDuration(using transitionContext: UIViewControllerContextTransitioning?) -> TimeInterval {
+        return 0.3
+    }
+    
+    func animateTransition(using transitionContext: UIViewControllerContextTransitioning) {
+        //  Get the two viewcontrollers
+        let from = transitionContext.viewController(forKey: .from)
+        let to = transitionContext.viewController(forKey: .to)
+        
+        guard
+            let fromView = from?.view,
+            let toView = to?.view else {
+                transitionContext.completeTransition(true)
+                return
+        }
+        
+        //  Get the container view - where the animation happens
+        let containerView = transitionContext.containerView
+        
+        //  Add the two viewcontroller views to the container
+        containerView.addSubview(fromView)
+        containerView.addSubview(toView)
+        
+        toView.alpha = 0.0
+        
+        UIView.animate(
+            withDuration: self.transitionDuration(using: transitionContext),
+            delay: 0,
+            options: [],
+            animations: {
+                toView.alpha = 1.0
+            }) { (finished) in
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/6f1c7836a2c0d74d84b1a9f67b523ee54a5b5f94/Headlines/src/ScreenSwitcher/FadeTransition.swift#L43"></span>Unused parameter “finished” in a closure should be replaced with _.</td>
    </tr>
  </tbody>
</table>